### PR TITLE
PMM-10382 Verify webConfigPlaceholder is generated on every Node exporter restart

### PR DESCRIPTION
**Pull request by peterSirotnak**

### CODE REVIEW

**Review by VoK0 — APPROVED** [review]
good 2 go 👍

File: tests/verifyPMMInventory_test.js [hunk 1]
```diff
@@ -213,3 +213,60 @@ Scenario(
...
+  async ({ I, pmmInventoryPage }) => {
+    I.amOnPage(pmmInventoryPage.url);
+    await I.waitForVisible(pmmInventoryPage.fields.agentsLink, 20);
+    I.click(pmmInventoryPage.fields.agentsLink);
+    await I.waitForVisible(pmmInventoryPage.fields.tableRow);
+    const nodeStatus = await I.grabTextFrom(pmmInventoryPage.fields.nodeExporterStatus);
+
+    assert.ok(nodeStatus.includes('RUNNING'), 'Node Exporter is not running');
+
+    const nodeId = await I.verifyCommand('docker exec pmm-server ls /tmp/node_exporter/agent_id/');
```
[the remark this answers is not in the document]
Comment (peterSirotnak) [reply tests/verifyPMMInventory_test.js:228]: Added coments.

**Comment by peterSirotnak** [comment]
> please attach nightly execution here

Nightly tag removed, we do not have docker on nighlty. 
UI tests after changes.
https://pmm.cd.percona.com/job/pmm2-ui-tests/20294/allure/#suites/fcc4843343a7d21212c0ada36d56bda3/b13bfbd84cce6e87/

File: tests/verifyPMMInventory_test.js [hunk 2]
```diff
@@ -213,3 +213,60 @@ Scenario(
     });
   },
 );
+
+Scenario(
+  'PMM-T1275 - Verify webConfigPlaceholder is generated on every Node exporter restart @inventory @nightly @exporters @tempTest',
```
[the remark this answers is not in the document]
Comment (peterSirotnak) [reply tests/verifyPMMInventory_test.js]: Left from developing test, removed.

File: tests/verifyPMMInventory_test.js [hunk 3]
```diff
@@ -213,3 +213,60 @@ Scenario(
...
+    const folderRestart = await I.verifyCommand(`docker exec pmm-server ls /tmp/node_exporter/agent_id/${nodeId}/`);
+
+    assert.ok(folderRestart.length > 0, 'webConfigPlaceholder was not recreated after restart');
+
+    await I.verifyCommand('docker exec pmm-server rm -r /tmp/node_exporter/');
+    let restartProcessId = nodeExporterRestart.split(/(\s+)/);
+
+    await I.verifyCommand(`sudo kill -9 ${restartProcessId[0]}`);
+    restartProcessId = await I.verifyCommand('pgrep node_exporter');
+    if (restartProcessId.length > 0) {
```
[the remark this answers is not in the document]
Comment (peterSirotnak) [reply tests/verifyPMMInventory_test.js]: comments added.

File: tests/verifyPMMInventory_test.js [hunk 4]
```diff
@@ -213,3 +213,60 @@ Scenario(
...
+
+    await I.verifyCommand('pgrep node_exporter', '', 'fail');
+    await I.wait(15);
+
+    const nodeExporterRestart = await I.verifyCommand('pgrep node_exporter');
+
+    assert.ok(nodeExporterRestart.length > 0, 'Node exporter is not restarted');
+    const folderRestart = await I.verifyCommand(`docker exec pmm-server ls /tmp/node_exporter/agent_id/${nodeId}/`);
+
+    assert.ok(folderRestart.length > 0, 'webConfigPlaceholder was not recreated after restart');
```
[the remark this answers is not in the document]
Comment (peterSirotnak) [reply tests/verifyPMMInventory_test.js]: changed, and added checks to verify if correct file is in folder

**Pull request merged**